web/src/js/__tests__/components/FlowTableSpec.tsx

```diff
@@ -1,57 +1,27 @@
 import * as React from "react";
-import renderer from "react-test-renderer";
-import { PureFlowTable as FlowTable } from "../../components/FlowTable";
-import TestUtils from "react-dom/test-utils";
-import { TFlow, TStore } from "../ducks/tutils";
-import { Provider } from "react-redux";
+import FlowTable, { PureFlowTable } from "../../components/FlowTable";
+
+import { render } from "../test-utils";
+import { select } from "../../ducks/flows";
 
 window.addEventListener = jest.fn();
 
 describe("FlowTable Component", () => {
-    let selectFn = jest.fn(),
```

Comment:
What exactly happened to those tests? They look completely different now?

Reply:
They were utterly wrong, now they are utterly right. 

![image](https://github.com/mitmproxy/mitmproxy/assets/1019198/5650a45d-b8e8-46c2-a610-4e29654f2942)